publish/BUILD.bazel

```diff
@@ -75,6 +75,9 @@ java_export(
     maven_coordinates = "dev.cel:cel:%s" % CEL_VERSION,
     pom_template = ":cel_pom",
     runtime_deps = ALL_TARGETS,
+    deploy_env = [
```

Comment:
- Move `deploy_env` below `name` parameter
- Could you pull this into a constant then make the same changes to other `java_export` targets?

Reply:
Thanks.

Establishing proper maven dependencies across artifacts is definitely something that needs to be addressed. I suspect this involves setting up a multi-stage workflow where dependent artifacts are published first. It's something I'm hoping to get around to soon.